as/plugins/org.jboss.ide.eclipse.as.core/jbosscore/org/jboss/ide/eclipse/as/core/publishers/PublishUtil.java

```diff
@@ -64,13 +64,6 @@ public static boolean deployPackaged(IModule[] moduleTree) {
 			return true;
 		} else if( moduleTypeId.equals(IWTPConstants.FACET_APP_CLIENT)) { 
 			return true;
-		} else if( moduleTypeId.equals(IWTPConstants.FACET_WEB_FRAGMENT)) {
```

Comment:
this logic should depend on the version of the server should it not ? I think there was a reason for why this was done?